apps/omg_eth/test/fixtures.exs

```diff
@@ -20,6 +20,8 @@ defmodule OMG.Eth.Fixtures do
 
   alias OMG.Eth.Configuration
   alias OMG.Eth.Encoding
+  alias OMG.Eth.RootChain.Abi, as: RootChainABI
```

Comment:
I am using this because in this file there is sth like `ABI.encode` where ABI is another library.
If we simply use alias, it will resulting in `ABI` and `Abi` and is pretty confusing though there is slight difference.